main/User_config.h

```diff
@@ -323,6 +323,7 @@ int lowpowermode = DEFAULT_LOW_POWER_MODE;
 //#define ZgatewayIR     "IR"       //ESP8266, Arduino,         Sonoff RF Bridge
 //#define ZgatewayLORA   "LORA"       //ESP8266, Arduino, ESP32
 //#define ZgatewayPilight "Pilight" //ESP8266, Arduino, ESP32
+//#define ZgatewayPilight_rawEnabled true // enables Pilight RAW return - switchable via MQTT
```

Comment:
Thanks for the PR. Could you add the macro in config_RF.h instead, there is a section dedicated to Pilight parameters. It could also be renamed to Pilight_rawEnabled
```
/*-------------------ESPPiLight topics & parameters----------------------*/
```

Reply:
Sure, done.